src/controllers/confirmation.controller.ts

```diff
@@ -0,0 +1,67 @@
+import { NextFunction, Request, Response } from "express";
+import {formatDateForDisplay} from "../client/date.formatter";
+import logger from "../logger";
+import {PTFCompanyProfile} from "../model/company.profile";
+import * as templatePaths from "../model/template.paths";
+import * as sessionService from "../services/session.service";
+import {ALREADY_SUBMITTED, COMPANY_NUMBER, COMPANY_PROFILE, SIGN_IN_INFO, USER_PROFILE} from "../session/keys";
+import {IUserProfile} from "../session/types";
+
+const ACCOUNTS_NEW_DEADLINE: number = 28;
+const CS_NEW_DEADLINE: number = 14;
```

Comment:
Could these have more descriptive variable names? Such as "CONFIRMATION_STATEMENT_EXTENSION_DAYS"
Make it clear the value is the number of days granted for that type of extension

Reply:
I'd prefer to avoid using the word "extension" in this project